Documentation/ApiOverview/ContentElements/CustomBackendPreview.rst

```diff
@@ -14,6 +14,65 @@ following sections describe how to achieve that.
 A preview renderer is used to facilitate (record) previews in TYPO3. This
 class is responsible for generating the preview and the wrapping.
 
+The default PreviewRenderer is :php:`StandardContentPreviewRenderer` and handles
+the core's built-in CTypes.
```

Comment:
```suggestion
the Core's built-in content types (field :sql:`CType` in table :sql:`tt_content`).
```

Reply:
Here is a list of special words and how we write and capitalize them in the docs: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/Glossary.html "The Core" is treated as propper noun and always written in Captitals unless we talk about "the core of the poodle"